packages/kbn-interpreter/src/plugin/types/datatable.js

```diff
@@ -78,5 +78,32 @@ export const datatable = () => ({
         },
       };
     },
+    pointseries: datatable => {
+      // datatable columns are an array that looks like [{ name: "one", type: "string" }, { name: "two", type: "string" }]
+      // rows look like [{ one: 1, two: 2}, { one: 3, two: 4}, ...]
+      const validFields = ['x', 'y', 'color', 'size', 'text'];
+      const columns = datatable.columns.filter(column => validFields.includes(column.name));
+      const rows = datatable.rows.map(row => pick(row, validFields));
+
+      return {
+        type: 'pointseries',
+        columns: columns.reduce((acc, column) => {
+          /* pointseries columns are an object that looks like this
+           * {
+           *   x: { type: "string", expression: "x", role: "dimension" },
+           *   y: { type: "string", expression: "y", role: "dimension" }
+           * }
+           */
+          acc[column.name] = {
+            type: column.type,
+            expression: column.name,
+            role: 'dimension',
```

Comment:
no-op: I don't think `role` is actually used anywhere. We could probably remove all occurrences of it if we wanted to.

Reply:
Yeah, I don't see it referenced anywhere, but removing it would be outside the scope of this PR.